### DISCUSSION TITLE
Android 源码分析 —— Handler、Looper 和 MessageQueue — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#217] [Announcements]
https://mazhuang.org/2018/06/11/handler-looper-massagequeue/ 

 本系列文章在 https://github.com/mzlogin/rtfsc-android 持(jing)续(chang)更(duan)新(geng)中，欢迎有兴趣的童鞋们关注。

**Comment by FSilence** [comment]
👍 非常喜欢这种提出问题然后带着问题解答的写做模式   学习了 之后也会自己尝试这种风格

**Comment by mzlogin** [comment]
> @FSilence
> 👍 非常喜欢这种提出问题然后带着问题解答的写做模式   学习了 之后也会自己尝试这种风格

很高兴获得认可。对阅读源码的长文写作风格，我也在摸索中，多交流多提意见 :)

**Comment by FSilence** [comment]
> @mzlogin
> > @FSilence
> > 👍 非常喜欢这种提出问题然后带着问题解答的写做模式   学习了 之后也会自己尝试这种风格
> 
> 很高兴获得认可。对阅读源码的长文写作风格，我也在摸索中，多交流多提意见 :)

自己也断断续续有尝试写点什么 但是总感觉很难表达 以后还要多向博主学习啊，， 话说 我才知道原来博主是搞android的，用了好久你的模板 还以为是搞web的，， 全能啊

**Comment by mzlogin** [comment]
> @FSilence
> > @mzlogin
> > > @FSilence
> > > 👍 非常喜欢这种提出问题然后带着问题解答的写做模式   学习了 之后也会自己尝试这种风格
> > 
> > 很高兴获得认可。对阅读源码的长文写作风格，我也在摸索中，多交流多提意见 :)
> 
> 自己也断断续续有尝试写点什么 但是总感觉很难表达 以后还要多向博主学习啊，， 话说 我才知道原来博主是搞android的，用了好久你的模板 还以为是搞web的，， 全能啊

惭愧惭愧……博而不精

**Comment by starifly** [comment]
@FSilence 年轻有为，再接再厉

**Comment by QingyiZhang** [comment]
您好，主页设置为https访问后，感觉css没有正确加载，界面显示不正常，求问如何解决？

**Comment by mzlogin** [comment]
> @QingyiZhang
> 您好，主页设置为https访问后，感觉css没有正确加载，界面显示不正常，求问如何解决？

把你的 _config.yml 里的 url 改为 https 开头看看

**Comment by zhangwufeng001** [comment]
厉害了，写得非常好

**Comment by spiolynn** [comment]
# 谢谢
